src/category_theory/limits/limits.lean

```diff
@@ -177,6 +178,15 @@ def iso_unique_cone_morphism {t : cone F} :
   { lift := λ s, (h s).default.hom,
     uniq' := λ s f w, congr_arg cone_morphism.hom ((h s).uniq ⟨f, w⟩) } }
 
+/--
+TODO: write me
```

Comment:
Perhaps easier to just let the linter fail, until you're ready to write it? :-)

Reply:
Oops, I intended to write this before making the PR! Sorry about that